shared/actions/signup.js

```diff
@@ -18,42 +18,32 @@ import * as Creators from './login/creators'
 import {navigateAppend, navigateTo} from '../actions/route-tree'
 
 import type {
-  CheckInviteCode,
   CheckUsernameEmail,
   CheckPassphrase,
   SubmitDeviceName,
   Signup,
   ShowPaperKey,
-  ShowSuccess,
-  ResetSignup,
-  RestartSignup,
   RequestInvite,
-  StartRequestInvite,
-  SignupWaiting,
 } from '../constants/signup'
-import type {NavigateAppend, NavigateTo} from '../constants/route-tree'
-import type {TypedAsyncAction, AsyncAction} from '../constants/types/flux'
 
-function nextPhase(): TypedAsyncAction<NavigateAppend> {
```

Comment:
i think we're likely not getting any value from thunks and defining what side effects it has